src/react/MODUKBody/MODUKBody.tsx

```diff
@@ -0,0 +1,13 @@
+'use client'
```

Comment:
Should I remove this should nextjs dev include this in their projcects

Reply:
Hmm seems to be React thing now: https://react.dev/reference/react/use-client

Problem is, it didn't quite work for me having it on our side and importing the ESM  for some reason... If you have to do it on the Next.js side you have to wrap MODUKBody in another component, otherwise you'd end up making the entire layout 'use client'...

Reply:
On further testing, it seems to work if I import `MODUKBody.js` directly, but not if I import `index.js`. So maybe we need another entrypoint that has `use client`? (And another CJS bundle :weary: )

Could be something for a separate issue because seems something relevant for other components too.